packages/viewer/src/components/viewer-view-cube/viewer-view-cube.css

```diff
@@ -54,7 +54,6 @@
    */
   --viewer-view-cube-triad-z-axis-color: var(--z-axis-color);
 
-  display: block;
```

Comment:
Was this meant to be included in this PR?

Reply:
Yep, there are two `display` properties that were set. `display: flex` is the one that should be used.